plugins/bean-validation/src/main/java/org/apache/struts/beanvalidation/validation/interceptor/BeanValidationInterceptor.java

```diff
@@ -148,9 +155,7 @@ private void addBeanValidationErrors(Set<ConstraintViolation<Object>> constraint
                     if (action instanceof ModelDriven && fieldName.startsWith(ValidatorConstants.MODELDRIVEN_PREFIX)) {
                         fieldName = fieldName.replace("model.", ValidatorConstants.EMPTY_SPACE);
                     }
-                    if (LOG.isDebugEnabled()) {
-                        LOG.debug("Adding field error [{}] with message [{}]", fieldName, validationError.getMessage());
-                    }
+                    LOG.debug("Adding field error [{}] with message [{}]", fieldName, validationError.getMessage());
```

Comment:
It seems we must revert deleting `if (LOG.isDebugEnabled()) {`. Please see [René Gielen comment](https://github.com/apache/struts/pull/200#discussion_r161456354)

Reply:
Same here, no additional cost to get `message`